### PR TITLE
Removing Kraken from hydra, (There are currently too many dependency…

### PR DESCRIPTION
**Pull request by sushilks**
…s in kraken with external/internal projects).

Will merge it back when a clean separation is possible.

Signed-off-by: Sushil Singh sushilks@gmail.com

### CODE REVIEW

File: src/main/python/hydra/lib/runtestbase.py [hunk 1]
```diff
@@ -500,6 +496,61 @@ def random_select_instances(self, app_name, cnt):
...
+        for app in self.appIdList:
+            self.__mt.wait_app_removal(app)
+
+    def get_mesos_slave_ips_attr(self, attr):
+        """
+        Get the ip of a mesos slave that matches the provided attribute
+        """
+        return self.__mesos.get_slave_ips_from_attribute(attr)
+
+    @staticmethod
```
Comment (kratos7) [inline src/main/python/hydra/lib/runtestbase.py:513]: I dont think this belongs here. Probably move it to common.

Comment (kratos7) [inline src/main/python/hydra/lib/runtestbase.py:513]: Or utility

File: src/unittest/python/runtest_local_tests.py [hunk 2]
```diff
@@ -4,60 +4,34 @@
...
-        l.info("test 2 launched")
+    def test_ip_block(self):
+        l.info("test ip block launched")
         self.TCP_IP = '127.0.0.1'
         self.TCP_PORT = 5005
         self.BUFFER_SIZE = 5
         self.t_exceptions = []
         self.h_threading = HThreading()
-        self.k_api = KrakenApi()
+        self.k_api = RunTestBase
```
Comment (kratos7) [inline src/unittest/python/runtest_local_tests.py:34]: self.k_api is now lying :)

File: src/main/python/hydra/lib/runtestbase.py (hunk 1 above)
Comment (sushilks) [inline src/main/python/hydra/lib/runtestbase.py:513]: Yes will do

**Pull request merged**